### PR TITLE
Miguel

### PR DESCRIPTION
**Pull request by zmazanv**
wizkid does it again

### CODE REVIEW

File: src/main/java/com/zipcodewilmington/froilansfarm/TomatoPlant.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.zipcodewilmington.froilansfarm;
+
+public class TomatoPlant extends Crop {
+    @Override
```
Comment (zmazanv) [inline src/main/java/com/zipcodewilmington/froilansfarm/TomatoPlant.java:4]: Add a blank line between line 3 and 4 to improve whitespace consistency across all files.

File: src/main/java/com/zipcodewilmington/froilansfarm/TomatoPlant.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    public Edible yield() {
+        if (hasBeenHarvested() && hasBeenFertilized()) {
+            return new Tomato();
+        } else {
+            return null;
+        }
+    }
+}
```
Comment (zmazanv) [inline src/main/java/com/zipcodewilmington/froilansfarm/TomatoPlant.java:12]: Add a blank line between lines 11 and 12 to improve whitespace consistency across all files.

File: src/main/java/com/zipcodewilmington/froilansfarm/TomatoPlant.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.zipcodewilmington.froilansfarm;
+
+public class TomatoPlant extends Crop {
+    @Override
+    public Edible yield() {
+        if (hasBeenHarvested() && hasBeenFertilized()) {
+            return new Tomato();
```
Comment (zmazanv) [inline src/main/java/com/zipcodewilmington/froilansfarm/TomatoPlant.java:7]: I would recommend changing `return new Tomato();` to `return (new Tomato());` to make this line more easily readable. By having the brackets, it is is easier to see at a glance the separation between the new object created that is return and the return keyword itself. This is up to preference, but it is a suggestion.

**Review by MMaldonado1017 — APPROVED** [review]
clean

**Review by MMaldonado1017 — APPROVED** [review]

**Review by chrisv0303 — APPROVED** [review]

**Pull request merged**